sw/device/lib/crypto/drivers/kmac.c

```diff
@@ -545,6 +549,14 @@ static status_t kmac_write_key_block(kmac_blinded_key_t *key) {
       KMAC_KEY_LEN_REG_RESVAL, KMAC_KEY_LEN_LEN_FIELD, key_len_enum);
   abs_mmio_write32(kKmacBaseAddr + KMAC_KEY_LEN_REG_OFFSET, key_len_reg);
 
+  // Write random words to the key registers.
```

Comment:
Whenever we are sending key shares through the bus, it may be worth considering randomizing the write order. There are some TODOs in the cyrptolib to tackle this later. 

Are we still planning to this at a later time?

Reply:
Yes, I am definitely planning to go through and randomize the order of a bunch of loops in the drivers now that the `random_order`stub is filled in! It's up next on my list actually, just wanted to do that as a separate pass for PR size reasons.